frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -762,13 +761,11 @@ const MultiStepWizard = (): JSX.Element => {
                 />
               ) : (
                 parameter && (
-                  <AlarmModal
-                    openModal={open && parameter.stateKey === stateKey}
+                  <AlarmModalContent
```

Comment:
Is there a specific reason we have `ValueModal` and `AlarmModalContent`, rather than both being `*Modal` or both being `*ModalContent`?